### PR TITLE
x32 Windows Gamedata Update for 4/18/2024 TF2 Update

### PR DESCRIPTION
**Pull request by CookieCat45**
This fixes the following Windows signatures that were changed in the update. It's possible there are more, but the server crashes when it's starting up instead of the extension complaining about missing gamedata, which I assume means it just needs to be updated and rebuilt.

Studio_LookupSequence
Studio_SelectWeightedSequence
SimThink_EntityChanged
CBaseEntity::PhysicsMarkEntitiesAsTouching
CBaseEntity::TakeDamage
CBaseAnimating::SetPoseParameter
CBaseAnimating::ResetSequence
CBaseAnimating::LookupPoseParameter

### CODE REVIEW

File: gamedata/cbasenpc.txt [hunk 1]
```diff
@@ -209,13 +209,13 @@
 			"Studio_LookupSequence"
 			{
 				"library"		"server"
-				"windows"		"\x55\x8B\xEC\x56\x8B\x75\x08\x85\xF6\x75\x2A\x33\xC0\x5E\x5D\xC3\x8B\xCE\xE8\x2A\x2A\x2A\x2A\x84\xC0\x74\x2A\x53"
+				"windows"		"\x55\x8B\xEC\x83\xEC\x14\x80\x3D\x2A\x2A\x2A\x2A\x00\x0F\x85\x2A\x2A\x2A\x2A\x56\x57\x6A\x00\xC6\x05\x2A\x2A\x2A\x2A\x01\xC7\x05\x2A\x2A\x2A\x2A\x2A\x2A\x2A\x2A\xC7\x05\x2A\x2A\x2A\x2A\x2A\x2A\x2A\x2A\xC7\x05\x2A\x2A\x2A\x2A\x2A\x2A\x2A\x2A\xE8\x2A\x2A\x2A\x2A\x83\xC4\x04\xA3\x2A\x2A\x2A\x2A\xE8\x2A\x2A\x2A\x2A\xFF\x35\x2A\x2A\x2A\x2A\xB9\x2A\x2A\x2A\x2A\xA3\x2A\x2A\x2A\x2A\xE8\x2A\x2A\x2A\x2A\x8B\xC8\xC1\xE1\x04\x2B\xC8\xA1\x2A\x2A\x2A\x2A\x8D\x34\x88\xC7\x46\x08\x2A\x2A\x2A\x2A\xC7\x06\x2A\x2A\x2A\x2A\xC7\x46\x04\x2A\x2A\x2A\x2A\xC7\x46\x0C\x05\x00\x00\x00"
```
Comment (KitRifty) [inline gamedata/cbasenpc.txt:212]: This doesn't seem correct. The function this points to looks to be an initialization function for some VScript bindings.

File: gamedata/cbasenpc.txt [hunk 2]
```diff
@@ -246,7 +246,7 @@
 			"CBaseEntity::PhysicsMarkEntitiesAsTouching"
 			{
 				"library"		"server"
-				"windows"		"\x55\x8B\xEC\x56\x8B\x75\x0C\x57\x8B\xF9\xB9\x2A\x2A\x2A\x2A\x56\xE8\x2A\x2A\x2A\x2A\xF3\x0F\x10\x46\x38\x8B\xCF\xF3\x0F\x11\x05\x2A\x2A\x2A\x2A\x8B\x46\x3C\xA3\x2A\x2A\x2A\x2A\x8B\x46\x40\xA3\x2A\x2A\x2A\x2A\x8B\x46\x44\xA3\x2A\x2A\x2A\x2A\x66\x8B\x46\x48\x66\xA3\x2A\x2A\x2A\x2A\x8B\x46\x4C\xA3\x2A\x2A\x2A\x2A\x8B\x46\x50\x8B\x75\x08\x56\xA3\x2A\x2A\x2A\x2A\xE8\x2A\x2A\x2A\x2A"
+				"windows"		"\x55\x8B\xEC\x56\x57\xFF\x75\x0C\x8B\xF1"
```
Comment (KitRifty) [inline gamedata/cbasenpc.txt:249]:
```suggestion
				"windows"		"\x55\x8B\xEC\x57\xFF\x75\x2A\x8B\xF9\xB9"
```

File: gamedata/cbasenpc.txt [hunk 3]
```diff
@@ -285,7 +285,7 @@
 			"CBaseAnimating::SetPoseParameter"
 			{
 				"library"		"server"
-				"windows"		"\x55\x8B\xEC\x8B\x45\x08\xD9\x45\x10"
+				"windows"		"\x55\x8B\xEC\x51\x8B\x45\x08\xD9\x45\x10"
```
Comment (KitRifty) [inline gamedata/cbasenpc.txt:288]:
```suggestion
				"windows"		"\x55\x8B\xEC\x51\x8B\x45\x2A\xD9\x45"
```

File: gamedata/cbasenpc.txt [hunk 4]
```diff
@@ -309,7 +309,7 @@
 			"CBaseAnimating::ResetSequence"
 			{
 				"library"		"server"
-				"windows"		"\x55\x8B\xEC\xA1\x2A\x2A\x2A\x2A\x53\x56\x57\x83\x78\x30\x00\x8B\xD9\x8B\x7D\x08"
+				"windows"		"\x55\x8B\xEC\x51\xA1\x2A\x2A\x2A\x2A\x53\x56\x57\x83\x78\x30\x00\x8B\xD9\x8B\x7D\x08"
```
Comment (KitRifty) [inline gamedata/cbasenpc.txt:312]:
```suggestion
				"windows"		"\x55\x8B\xEC\x51\xA1\x2A\x2A\x2A\x2A\x53\x56\x57\x83\x78\x2A\x00\x8B\xD9\x8B\x7D"
```

**Review by KitRifty — CHANGES_REQUESTED** [review]
Hey, thanks for the update!

I was in the process of fixing some of the Windows signatures as well. Most of the signatures are correct (with some just needing some minor adjustments). `Studio_LookupSequence` needs to be looked over again, though.

File: gamedata/cbasenpc.txt (hunk 1 above)
Comment (KitRifty) [reply gamedata/cbasenpc.txt:212]: Found it, use this one:
```suggestion
				"windows"		"\x55\x8B\xEC\x56\x8B\x75\x2A\x85\xF6\x0F\x84\x2A\x2A\x2A\x2A\x8B\xCE"
```

Comment (CookieCat45) [reply gamedata/cbasenpc.txt:212]: Seems to be a bit of an oddball on Windows, maybe this one?
\x55\x8B\xEC\x56\x8B\x75\x08\x85\xF6\x0F\x84\x2A\x2A\x2A\x2A\x8B\xCE

Comment (CookieCat45) [reply gamedata/cbasenpc.txt]: Oops, forgot to click the button.

**Review by KitRifty — APPROVED** [review]
There are still some more signatures + the extension needs to be updated, but I'll pull this in for now and I'll add with my own changes in another PR. Thanks for the contribution!

**Pull request merged**